mautic/mautic
# "IF NOT EXISTS" added to all "CREATE TABLE" migrations

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5496, [Slack report](https://mautic.slack.com/archives/C02HU8BUT/p1516163740000245)
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When fresh Mautic instance is installed it creates all DB tables from entities. When upgrade is executed and all the migrations kick in they try to create some tables again. But they exist already and it will lead to a SQL error described in linked reports.

This PR adds `IF NOT EXISTS` statement into each `CREATE TABLE` query so it will skip creating tables which already exist.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Install fresh Mautic from git.
2. Run `app/console doctrine:migrations:migrate` - the errors should appear

#### Steps to test this PR:
1. Checkout this PR
2. Run the migrations again, they all should pass.
